composer.json

```diff
@@ -12,7 +12,7 @@
     ],
     "require": {
         "php": ">=5.4",
-        "php-http/httplug": "dev-utility_separation",
+        "php-http/httplug": "dev-master@dev",
```

Comment:
0.2 is fine for now. At least I don't really care, until httplug 1.0alpha is out :stuck_out_tongue: